assets/src/stories-editor/components/custom-video-block-edit.js

```diff
@@ -137,8 +137,8 @@ class CustomVideoBlockEdit extends Component {
 			this.setState( { extractingPoster: true } );
 
 			uploadVideoFrame( { id, src } )
-				.then( ( posterUrl ) => {
-					setAttributes( { poster: posterUrl } );
+				.then( ( fileObj ) => {
+					setAttributes( { poster: getPosterImageFromFileObj( fileObj ) } );
```

Comment:
Actually... Should this be applied for the Video block as well? Or just the background video?

Reply:
@miina you wrote this comment on the video block, soo... 🤔

Reply:
I know :) I wanted to ask with this if this actually should be applied to the Video block as well? In other words, is it intended that the poster image of a video block has the size based on the Page's poster image's recommended size? Shouldn't the poster image be based on the video's size instead?

Reply:
I guess the goal was to limit the size. So feel free to ignore!